recipes/gdk-pixbuf/all/test_package/test_package.c

```diff
@@ -0,0 +1,8 @@
+#include <stdio.h>
+#include "gdk-pixbuf/gdk-pixbuf.h"
+
+int main()
+{
+    printf("gdk-pixbuf version %s %d.%d.%d\n", gdk_pixbuf_version, gdk_pixbuf_major_version, gdk_pixbuf_minor_version, gdk_pixbuf_micro_version);
```

Comment:
Could you add a method? Just to test the linkage. Here you are consuming variables from a header.

Reply:
these are external variables, so they link the same way functions do right ?

Reply:
True! I thought they are macros!

Reply:
I did it anyway ;)